deployment/deployment-config.ts

```diff
@@ -15,6 +15,7 @@ export interface DeploymentConfigProperties {
   vpnIpSetArn?: string;
   vpnIpv6SetArn?: string;
   brokerString: string;
+  kafkaSubnetIds: string;
```

Comment:
Should this be kafka authororized subnet ids?  This isn't actually Kafka, but the subnets allowed to get to kafka.

Reply:
Changed to kafkaAuthorizedSubnetIds